### PR TITLE
Expose class_id parameter on subclasses of SensitivitySpecificityBase

### PR DESCRIPTION
**Pull request by dwyatte**
This PR addresses https://github.com/tensorflow/tensorflow/issues/46017 by simply exposing the `class_id` parameter on all subclasses of `SensitivitySpecificityBase`.

I've added tests to confirm this works on 3-way multi-label data and also added a guard for a test from `tensorflow/python/keras:metrics_confusion_matrix_test` that wasn't working locally due to requiring scipy (seems this pattern is used elsewhere in the codebase for running tests locally).

### CODE REVIEW

**Review by fchollet — APPROVED** [review]
LGTM, thank you
